lotus-soup/testkit/role_drand.go

```diff
@@ -127,16 +128,18 @@ func (dr *DrandInstance) RunDKG(nodes, thr int, timeout string, leader bool, lea
 
 func (dr *DrandInstance) Halt() {
 	dr.t.RecordMessage("drand node #%d halting", dr.t.GroupSeq)
-	dr.daemon.StopBeacon()
```

Comment:
This is a major change in this PR. Using `StopBeacon` to simulate Drand outage is probably not desirable, as it doesn't really test any possible real world scenario - we are stopping the beacon, but GRPC connections stay open, which essentially results in blocked requests once Drand is resumed.

Therefore I am changing to `Stop` which also closes any connections and stops HTTP APIs.